src/ts/errorutils.ts

```diff
@@ -73,7 +73,7 @@ export namespace ErrorUtils {
   };
 
   export const defaultWebErrorMessageTranslator: ErrorMessageTranslator = (error: ErrorEvent) => {
-    const errorMessage = ErrorUtils.defaultErrorMessages[error.code];
+    const errorMessage = error.message ?? ErrorUtils.defaultErrorMessages[error.code];
```

Comment:
Are we sure that all our error messages are formatted properly? I think this is more important when displaying it in the UI than when logging to the console.

And also that they don't include any internal knowledge that shouldn't be public OR information that is not relevant to the end user.

Reply:
These error messages are brand new, so nothing unexpected should be exposed through these. Whether it's worth exposing them through the UI is a good question, we have a ticket to do it but I'll rediscuss with the team.

Reply:
You made an excellent point and we also decided that it's not worth it, as the message won't be more interesting to the user at all: quite the opposite. Thanks a lot for the input, we did not think this through hard enough 😅